Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/CursorModifier.cs

```diff
@@ -243,7 +243,7 @@ void IMixedRealityFocusChangedHandler.OnBeforeFocusChange(FocusEventData eventDa
         #region MonoBehaviour Implementation
 
         private void OnValidate()
-        {
+        {   // This is an appropriate use of OnValidate
```

Comment:
I'm not super convinced there's a difference between this case and any of the others. In both cases, we're calling `GetComponent`. One just serializes the data while the other checks that it exists.

Serializing data at edit time is a valid thing to do. What's inappropriate about the other uses?

Reply:
Yeah it's pretty subjective, I could go either way on this one. I think what tipped the scales here was:
- you can't use RequireComponent because you need a _generic_ collider
- a custom inspector seems like total overkill for such a simple check
- this feels like an issue we should raise in-editor? This is the vaguest criteria.